<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>回显模式</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEcho"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnEchoPrint">
          <property name="text">
           <string>输出内容</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnEchoNormal">
          <property name="text">
           <string>正常模式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEchoPassword">
          <property name="text">
           <string>密码模式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEchoNo">
          <property name="text">
           <string>无回显</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEchoPasswordEchoOnEdit">
          <property name="text">
           <string>失去焦点变密码模式</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>对齐方式</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineAlign">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 138, 221);</string>
        </property>
        <property name="text">
         <string>我是一段文本</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btnAlignLeft">
          <property name="text">
           <string>左对齐</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAlignCenter">
          <property name="text">
           <string>居中对齐</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAlignRight">
          <property name="text">
           <string>右对齐</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>读写控制</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineAccess"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnAccessPrint">
          <property name="text">
           <string>输出内容</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="btnAccessReadWrite">
          <property name="text">
           <string>可读可写</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAccessReadOnly">
          <property name="text">
           <string>只读</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAccessDisabled">
          <property name="text">
           <string>失能</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAccessEnabled">
          <property name="text">
           <string>使能</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>格式控制</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineMask"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="btnMaskNone">
          <property name="text">
           <string>无控制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMaskPhone">
          <property name="text">
           <string>手机号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMaskLicense">
          <property name="text">
           <string>密钥</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>信号槽</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>textChange信号：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLineEdit" name="lineAdd1"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineAdd2"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineSum"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>editingFinished信号：</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLineEdit" name="lineSub1"/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineSub2"/>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineResult"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
